### ISSUE TITLE
Guru Meditation - HTML Response Received

### ISSUE THREAD

**Issue by bast1en** [pspete/psPAS#533]
Hi,

I'm using the following test code to get accounts in a Safe:

```
$cred = Get-Credential
New-PASSession -IdentityTenantURL https://abc123.id.cyberark.cloud -PrivilegeCloudURL https://blabla.cyberark.cloud -Credential $Cred -IdentityUser
Get-PASAccount -SafeName "TheSafeName"
```

Authentication with CyberArk Identity seems OK (with Mobile authn for MFA), as I don't have an error at this tage.
However when I add the "Get-PASAccount" command, then I get this error "Guru Meditation - HTML Response Received".

Thanks for your help.

**Comment by bast1en** [comment]
Get-IDSession works fine, so authn is OK I think

![image](https://github.com/pspete/psPAS/assets/18058700/0afce769-c0fd-40f6-85eb-aec0d2b85a73)

**Comment by pspete** [comment]
Had some time to look into this and can replicate the issue.
Currently no idea why it is happening, but using the command/method in previous response does not exhibit the behaviour.
So continue to suggest using `New-PASSession -tenantSubdomain yourPCloudTenant -Credential $cred -IdentityUser` until the issue is fixed

**Comment by pspete** [comment]
Correction - this is due to an incorrect value for privilege cloud URL being specified.

Refer to the docs here: https://pspas.pspete.dev/commands/New-PASSession#example-30

@bast1en , in your example code the value provided for `PrivilegeCloudURL` needs to be updated as per the example:

```powershell
$cred = Get-Credential
New-PASSession -IdentityTenantURL https://abc123.id.cyberark.cloud -PrivilegeCloudURL https://blabla.privilegecloud.cyberark.cloud -Credential $Cred -IdentityUser
Get-PASAccount -SafeName "TheSafeName"
```

@varuns29 - am assuming you just need to follow the example in the docs also?

**Comment by bast1en** [comment]
Hello @pspete, I had a chance to test again. 
Although authentication works (including MFA), the rest is not working for me:

The code:
```
$cred = Get-Credential
New-PASSession -IdentityTenantURL https://abc1234.id.cyberark.cloud -PrivilegeCloudURL https://mypcloudtenant.privilegecloud.cyberark.cloud -Credential $Cred -IdentityUser

Write-Host -ForegroundColor Yellow "- Trying Get_IDSession:"
Get-IDSession
Write-Host -ForegroundColor Yellow "- End."
Write-Host -ForegroundColor Yellow "- Trying Get-PASAccount -SafeName ""TheSafeName"":"
Get-PASAccount -SafeName "TheSafeName"
Write-Host -ForegroundColor Yellow "- End."
Write-Host -ForegroundColor Yellow "- Closing session with Close-PASSession:"
Close-PASSession
Write-Host -ForegroundColor Yellow "- End."
```

The result:
![image](https://github.com/pspete/psPAS/assets/18058700/a2537ffc-e97f-4bbd-b656-bc34dd02ceb3)

**Comment by marc-fom** [comment]
Hi all,

I battled with the same issue for days (with pauses in between) and today I finally found what I did wrong.

Pete's suggestion is the right one, the root cause is using the wrong root baseURL(s).
I somehow reused for -PrivilegeCloudURL the Shared Services pCloud web URL that I use interactively to access my cloud PVWA but psPAS is expecting something else and I realized that after looking at my own .ps1 provisioning scripts that don't leverage psPAS module (all written from scratch), and they use a different baseURL for accessing pCloud REST API.

In summary, in my case this will cause the Guru meditation error:
New-PASSession -IdentityTenantURL 'https://AABCD.id.cyberark.cloud' -PrivilegeCloudURL 'https://mytenant.**cyberark.cloud**' -Credential $Cred -ServiceUser

This will succeed:
New-PASSession -IdentityTenantURL 'https://AABCD.id.cyberark.cloud' -PrivilegeCloudURL 'https://mytenant.**privilegecloud.cyberark.cloud**' -Credential $Cred -ServiceUser

Hope this helps.

**Comment by bast1en** [comment]
Hi @marc-fom , thanks for the update. Unfortunately it's not working for me.

As you can see in my message, that's also the command I use, except that I use -IdentityUser instead of -ServiceUser. 
Maybe that's the difference ?

@pspete any idea why I get this 'Exception calling ".ctor" with "4" argument(s) ... ' error ?
